package.json

```diff
@@ -130,6 +131,8 @@
     "build": "node --max-old-space-size=3072 ./scripts/build/build.js",
     "build-docs": "node ./scripts/build-docs.js",
     "check-deps": "node ./scripts/check-deps.js",
-    "spellcheck": "npx -p cspell@4.0.31 cspell {bin,scripts,src}/**/*.js {docs,website/blog,changelog_unreleased}/**/*.md"
+    "spellcheck": "npx -p cspell@4.0.31 cspell {bin,scripts,src}/**/*.js {docs,website/blog,changelog_unreleased}/**/*.md",
+    "svgo": "svgo --pretty --indent=2 --precision=1",
+    "svgo:all": "find website | grep '\\.svg$' | xargs -Iz -n 1 yarn svgo z"
```

Comment:
Can't work for non linux users, so contributing from windows will be not easy :smile:

Reply:
Could this command be added to CI? This will helps us ensure

1. it works on Windows
2. There are no produced git diffs following its execution (i.e. svgs are optimised by the committer)

Reply:
`svgo:all` will run once.. and who ever else wants to add new images could run `yarn svgo path/to/image.svg`

Reply:
I removed it..